### PR TITLE
Feature 17 tasks

### PR DESCRIPTION
**Pull request by kaeline**
https://github.com/Metatavu/lipsanen-project-management-spec/issues/17

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -746,6 +1025,90 @@ components:
...
+          format: date
+          type: string
+          description: Can be edited only within proposal
+        endDate:
+          format: date
+          type: string
+          description: Can be edited only within proposal
+        milestoneId:
+          type: string
+          format: uuid
```
Comment (kaeline) [inline swagger.yaml:1325]: add list of TaskConnection where current task is parent or child

File: swagger.yaml [hunk 2]
```diff
@@ -746,6 +1025,90 @@ components:
...
+        metadata:
+          $ref: '#/components/schemas/Metadata'
+
+    TaskConnection:
+      type: object
+      description: Task connection
+      required:
+        - fromTaskId
```
Comment (kaeline) [inline swagger.yaml]: source/target

File: swagger.yaml [hunk 3]
```diff
@@ -746,6 +1025,90 @@ components:
...
+        type:
+          $ref: '#/components/schemas/TaskConnectionType'
+        metadata:
+          $ref: '#/components/schemas/Metadata'
+
+    TaskConnectionType:
+      type: string
+      description: Task connection type
+      enum:
+        - FINISH_TO_START
```
Comment (kaeline) [inline swagger.yaml:1372]: finish to finish

File: swagger.yaml [hunk 4]
```diff
@@ -21,6 +21,285 @@ paths:
...
+      responses:
+        "204":
+          description: No content
+        "default":
+          description: Invalid request was sent to the server
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+  /v1/tasks:
```
Comment (kaeline) [inline swagger.yaml]: under milestones

File: swagger.yaml [hunk 5]
```diff
@@ -375,6 +376,357 @@ paths:
...
+          description: No content
+        "default":
+          description: Invalid request was sent to the server
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+  /v1/projects/{projectId}/milestones/{milestoneId}/tasks:
+    get:
+      operationId: listMilestoneTasks
```
Comment (lauriLukkarinen) [inline swagger.yaml]: Should the operations be [method]ProjectMilestoneTasks?

File: swagger.yaml [hunk 6]
```diff
@@ -931,6 +1296,84 @@ components:
         metadata:
           $ref: '#/components/schemas/Metadata'
 
+
```
Comment (lauriLukkarinen) [inline swagger.yaml]:
```suggestion
```

**Review by lauriLukkarinen — APPROVED** [review]

**Pull request merged**